codes/quantum/qubits/subsystem/subsystem-gen-shor.yml

```diff
@@ -0,0 +1,46 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+code_id: subsystem-gen-shor
+physical: qubits
+logical: qubits
+
+
+name: 'Subsystem-Generalized Shor Code'
+introduced: '\cite{arXiv:quant-ph/0610088v2}'
+
+
+description: |
+  For every generalized Shor code, there is a subsystem code which has the same parameters as the subspace version, but requires significantly fewer stabilizer measurements, resulting in a much simpler error recovery routine.
+
+  This code is constructed from two classical linear codes, \(\mathcal{C}_1 = [n_1, k_1, d_1]$ and $\mathcal{C}_2 = [n_2, k_2, d_2]\). It has considerable advantages over the subspace code when it comes to the complexity of error recovery routine, providing a quadratic savings in the number of stabilizers which need to be measured.
+
+protection: 'Detecting single qubit errors of weight \(min(d_1,d_2)\), this code corrects bit flip and phase flip errors modulo the subsystem structure.'
+
+features:
+
+  encoders:
```

Comment:
This isn't really a description of the encoder, but rather a description of the code construction. I think it should be moved to the description part and simplified. As the code is CSS, the general method for encoding CSS codes can be used for this family. Also I don't like 'P' as notation for parity-check matrices; H is more standard.

Reply:
Please follow notation of css.yml